tutorials/CMakeLists.txt

```diff
@@ -53,14 +53,20 @@ find_package(Vitis)
 find_package(Python3 COMPONENTS Interpreter)
 
 # Look for LibXAIE in the sysroot.  Depending on the sysroot, this might live in multiple locations.
-set(LibXAIE_ROOT ${VitisSysroot}/usr ${VitisSysroot}/opt/xaiengine ${VitisSysroot}/opt/xaienginev2 STRING "Default libxaie location.")
-find_package(LibXAIE)
-
-if(LibXAIE_FOUND)
-message(STATUS "Using LibXAIE in: ${XILINX_XAIE_DIR}")
+if(NOT extraAieCcFlags)
+  set(LibXAIE_ROOT ${VitisSysroot}/usr ${VitisSysroot}/opt/xaiengine ${VitisSysroot}/opt/xaienginev2)
+  find_package(LibXAIE)
+  if(LibXAIE_FOUND)
+    message(STATUS "Using LibXAIE in sysroot: ${XILINX_XAIE_DIR}")
+  else()
+    message(STATUS "LibXAIE NOT FOUND in ${VitisSysroot}: Skipping some tests")
+  endif()
 else()
-message(STATUS "LibXAIE NOT FOUND: Skipping some tests")
+  set(LibXAIE_ROOT ${CMAKE_INSTALL_PREFIX}/runtime_lib/libxaie)
```

Comment:
Match library name

Reply:
Done. matched to xaiengine until aie-rt gets updated.